cmd/templ/generatecmd/main.go

```diff
@@ -44,8 +44,8 @@ type Arguments struct {
 	GenerateSourceMapVisualisations bool
 	IncludeVersion                  bool
 	IncludeTimestamp                bool
-	// PPROFPort is the port to run the pprof server on.
-	PPROFPort int
+	PPROFPort                       int // PPROFPort is the port to run the pprof server on.
```

Comment:
Looks like the comment has been accidentally moved? In Go, comments above the struct field are automatically included in text editor autocomplete documentation. Moving the comment to the side breaks that behaviour.

Reply:
Oops! Did not know that. I unfixed my ocd fix.